src/throttler/throttler.ts

```diff
@@ -46,7 +46,7 @@ export interface ThrottlerStats {
   elapsed: number;
 }
 
-interface TaskData<T, R> {
+interface TaskData<T> {
```

Comment:
Hmmm, looks like this may have been a typo & possibly should be used? `TaskData.wait.resolve` is of type `(R: any) => void`- i suspect that this should actually be something like `(val: R) => void`

Reply:
You... are... right! This lead me down a weird path of fixing a handful of types.